authorization/group/type.go

```diff
@@ -0,0 +1,5 @@
+package group
+
+const (
+	IdentityResourceTypeGroup = "identity/group"
```

Comment:
I was thinking it might be best to have a single authorization/constants type which contains all these, otherwise it's going to get real tedious importing every single one from a different location.

Reply:
I could do a `authorization/constants/constants.py`  and import that everywhere.
Since these are just string constants and will not contain any `struct` definitions ( thereby no business/service/model logic ).

What do you think?

Reply:
Is there a point to putting it in its own directory when there's just one source file?  I would think it makes more sense keeping it in the authorization package, especially since there won't be much else in that package besides this?

Reply:
If there is some "local" constant then it should be in the corresponding sub package like /authorization/group/group.go
But it makes sense to keep the shared/common constants like *TypeGroup in one place. And I would use /authorization/authorization.go for such constants. Not constants.go.